GameServer/Logic/GameObjects/CollisionHandler.cs

```diff
@@ -1,433 +1,60 @@
-﻿using LeagueSandbox.GameServer.Core.Logic;
+﻿using System;
+using LeagueSandbox.GameServer.Core.Logic;
 using LeagueSandbox.GameServer.Logic.Maps;
-using System;
 using System.Collections.Generic;
-using System.Numerics;
+using System.Diagnostics;
 
 namespace LeagueSandbox.GameServer.Logic.GameObjects
```

Comment:
The changes here are somewhat questionable, but since the collision system will be re-implemented later anyway, I'd say this is alright.